packages/validation/src/research-output-guards.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  ResearchOutputResponse,
+  ResearchOutputPostRequest,
+} from '@asap-hub/model/src/research-output';
+
+export const isTeamResearchOutput = (
```

Comment:
This is not how you write type guards though
```suggestion
export const isTeamResearchOutput = (
   researchOutputData: ResearchOutputResponse | ResearchOutputPostRequest,
): researchOutputData is ResearchOutputResponse => {
```
is how you would type it - except I don't understand what you are doing here. Why are you deciding between a PostRequest and a Response? 

What you should do is declare two types either for the Response or Post request (I think you want it for the Response). Your `ResearchOutputTeamResponse` should contain no references to WGs whereas  `ResearchOutputWorkingGroupResponse` would have one. Then you can write this typeguard this way
```typescript
export const isResearchOutputTeamResponse = (
   researchOutputData: ResearchOutputTeamResponse | ResearchOutputWorkingGroupResponse,
): researchOutputData is ResearchOutputTeamResponse => !!researchOutputData.workingGroups.length
```